docs/pages/api-docs/data-grid/data-grid.md

```diff
@@ -163,13 +163,13 @@ Api of the `components` props of type `GridSlotsComponent`
 |  | <span class="prop-name">.MuiDataGrid-window</span> | Styles applied to the window element.|
 |  | <span class="prop-name">.MuiDataGrid-viewport</span> | Styles applied to the viewport element.|
 | <span class="prop-name">row</span> | <span class="prop-name">.MuiDataGrid-row</span> | Styles applied to the row element.|
-|  | <span class="prop-name">.Mui-selected</span> | Styles applied to the selected row element.|
-|  | <span class="prop-name">.MuiDataGrid-cellWithRenderer</span> | Styles applied to the customized cell element.|
+|  | <span class="prop-name">.Mui-selected</span> | Pseudo-class applied to the selected elements.|
+|  | <span class="prop-name">.MuiDataGrid-cell--withRenderer</span> | Styles applied to the customized cell element.|
 |  | <span class="prop-name">.MuiDataGrid-withBorder</span> | Styles applied to the cell element that has right border displayed.|
-|  | <span class="prop-name">.MuiDataGrid-cellLeft</span> | Styles applied to the aligned left cell element.|
-|  | <span class="prop-name">.MuiDataGrid-cellRight</span> | Styles applied to the aligned right cell element.|
-|  | <span class="prop-name">.MuiDataGrid-cellCenter</span> | Styles applied to the centered cell element.|
-|  | <span class="prop-name">.MuiDataGrid-footer</span> | Styles applied to the footer element.|
+|  | <span class="prop-name">.MuiDataGrid-cell--textLeft</span> | Styles applied to the aligned left cell element.|
+|  | <span class="prop-name">.MuiDataGrid-cell--textRight</span> | Styles applied to the aligned right cell element.|
+|  | <span class="prop-name">.MuiDataGrid-cell--textCenter</span> | Styles applied to the centered cell element.|
+|  | <span class="prop-name">.MuiGridFooterContainer</span> | Styles applied to the footer element.|
```

Comment:
If it's standalone it doesn't need to appear here I think.

```suggestion
```

Reply:
Where would it appear then. I still think we need to keep them at least for now because there won't be documented anywhere. Also, the 'standalone' term is dangerous because we are implying that you would be able to use that component by itself where the truth is that its value is closely tied to the grid and although you could use it it wouldn't be beneficial.

Reply:
As suggested in https://github.com/mui-org/material-ui-x/pull/1872#discussion_r651695266, there'll be no more standalone components so `.MuiGridFooterContainer` -> `.MuiDataDrid-footerContainer`.